src/App.js

```diff
@@ -1,23 +1,71 @@
-import logo from './logo.svg';
+// import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import contacts from "./contacts.json";
 
 function App() {
+
+  const firstFive = contacts.slice(0, 5);
+  const[contactsArray, setContactsArray] = useState(firstFive)
+
+  const randomizeContacts = () => {
```

Comment:
The `addRandom()` should add a random contact to the displayed contacts list, the logic to randomize the contacts work => But if the array is bigger or you need a real random array you should use another algorithm to do that one